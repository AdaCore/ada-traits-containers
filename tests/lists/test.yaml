description: 'Basic test for lists'
driver: 'build_and_exec'
