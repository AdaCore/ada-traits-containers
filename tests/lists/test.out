------- Definite Bounded --------------
list, element loop => 1
list, element loop => 2
list, element loop => 3
list, element loop => 4
list, cursor loop => 1
list, cursor loop => 2
list, cursor loop => 3
list, cursor loop => 4
assigned list, element loop => 1
assigned list, element loop => 2
assigned list, element loop => 3
assigned list, element loop => 4
------- Definite Bounded (limited) --------------
list, element loop => 1
list, element loop => 2
list, element loop => 3
list, element loop => 4
list, cursor loop => 1
list, cursor loop => 2
list, cursor loop => 3
list, cursor loop => 4
assigned list, element loop => 1
assigned list, element loop => 2
assigned list, element loop => 3
assigned list, element loop => 4
------- Definite Unbounded --------------
list, element loop => 1
list, element loop => 2
list, element loop => 3
list, element loop => 4
list, cursor loop => 1
list, cursor loop => 2
list, cursor loop => 3
list, cursor loop => 4
assigned list, element loop => 1
assigned list, element loop => 2
assigned list, element loop => 3
assigned list, element loop => 4
------- Indefinite Unbounded --------------
list, element loop => 1
list, element loop => 2
list, element loop => 3
list, element loop => 4
list, cursor loop => 1
list, cursor loop => 2
list, cursor loop => 3
list, cursor loop => 4
assigned list, element loop => 1
assigned list, element loop => 2
assigned list, element loop => 3
assigned list, element loop => 4
------- Indefinite Unbounded SPARK --------------
list, element loop => 1
list, element loop => 2
list, element loop => 3
list, element loop => 4
list, cursor loop => 1
list, cursor loop => 2
list, cursor loop => 3
list, cursor loop => 4
assigned list, element loop => 1
assigned list, element loop => 2
assigned list, element loop => 3
assigned list, element loop => 4
